test/common/http/filter_manager_test.cc

```diff
@@ -169,7 +172,21 @@ TEST_F(FilterManagerTest, SendLocalReplyDuringDecodingGrpcClassiciation) {
   EXPECT_CALL(filter_manager_callbacks_, resetIdleTimer());
   EXPECT_CALL(filter_manager_callbacks_, encodeHeaders(_, _));
   EXPECT_CALL(filter_manager_callbacks_, endStream());
+
   filter_manager_->decodeHeaders(*grpc_headers, true);
+
+  ASSERT_TRUE(filter_manager_->streamInfo().filterState()->hasData<LocalReplyOwnerObject>(
```

Comment:
Can you put these checks into a helper method in the test class? It's fairly repetitive.